.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        php-versions: [ '8.0' ]
+        coverage-driver: [ 'pcov', 'xdebug3' ]
+
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v2
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: Install dependencies
```

Comment:
In the logs of this job i can see: `Warning: Unexpected input(s) 'composer_version', valid inputs are ['version', 'php_version', 'php_extensions', 'command', 'only_args', 'interaction', 'dev', 'progress', 'quiet', 'args', 'ssh_key', 'ssh_key_pub', 'ssh_domain', 'working_dir']`